recipe_scrapers/maangchi.py

```diff
@@ -43,7 +42,12 @@ def instructions(self):
         )
 
     def ratings(self):
-        return self.schema.ratings()
```

Comment:
Ok, interesting: so this is a case where `aggregateRating` information _is_ still available in the page metadata, but it has moved into an `@id`-based reference to another entity.

I've fixed #908 to track this.  _Maybe_ we can move ahead with this batch before resolving that -- personally I'd prefer if we could fix it separate to these changes (so that the commit history and reasoning for the changes is clearer).

Reply:
I'll take a look at the other issue when i have some time this weekend!

Reply:
I marked this as resolved, but: I think we can simplify this to `self.schema.ratings()` with your fix from #913 @jknndy?

Reply:
Yep! Fixed now, I'll get started on the last batch sometime this weekend!